integration/tests/params.go

```diff
@@ -0,0 +1,163 @@
+// +build integration
+
+package tests
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/weaveworks/eksctl/integration/runner"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/utils/names"
+)
+
+// Params groups all test parameters.
+type Params struct {
+	EksctlPath string
+	Region     string
+	Version    string
+	// Flags to help with the development of the integration tests
+	clusterNamePrefix string
+	ClusterName       string
+	DoCreate          bool
+	DoDelete          bool
```

Comment:
nit: how about renaming this to `deleteClusterAfterTest`? too long?

Reply:
It could be used for more than that. What about one of these?
```go
SetUp bool
TearDown bool
```
```go
DoSetUp bool
DoTearDown bool
```
```go
ShouldSetUp bool
ShouldTearDown bool
```

Reply:
Yeah, that also works. I think something saying "create", "destroy", "remove", "delete" is perhaps more clear but if you think there are more things to do in those steps then those names look good

Reply:
It could also be "reversed", e.g.:
```go
SkipSetUp bool
SkipTearDown bool
```
or
```go
SkipCreate bool
SkipDelete bool
```
Any preference?

Reply:
Yeah, I think I like the last ones more :)

Reply:
Fixed. 👍